test/integration/narrow/routes/endemics/vet-rcvs.test.js

```diff
@@ -0,0 +1,142 @@
+const getEndemicsClaimMock = require('../../../../../app/session').getEndemicsClaim
+const { rcvs: rcvsErrorMessages } = require('../../../../../app/lib/error-messages')
+jest.mock('../../../../../app/session')
+
+describe('Vet name test', () => {
+  const auth = { credentials: {}, strategy: 'cookie' }
+  const url = '/claim/endemics/vet-rcvs'
+
+  beforeAll(() => {
+    getEndemicsClaimMock.mockImplementation(() => { return { typeOfLivestock: 'pigs' } })
+
+    jest.mock('../../../../../app/config', () => {
+      const originalModule = jest.requireActual('../../../../../app/config')
+      return {
+        ...originalModule,
+        authConfig: {
+          defraId: {
+            hostname: 'https://tenant.b2clogin.com/tenant.onmicrosoft.com',
+            oAuthAuthorisePath: '/oauth2/v2.0/authorize',
+            policy: 'b2c_1a_signupsigninsfi',
+            redirectUri: 'http://localhost:3000/apply/signin-oidc',
+            clientId: 'dummy_client_id',
+            serviceId: 'dummy_service_id',
+            scope: 'openid dummy_client_id offline_access'
+          },
+          ruralPaymentsAgency: {
+            hostname: 'dummy-host-name',
+            getPersonSummaryUrl: 'dummy-get-person-summary-url',
+            getOrganisationPermissionsUrl: 'dummy-get-organisation-permissions-url',
+            getOrganisationUrl: 'dummy-get-organisation-url'
+          }
+        },
+        endemics: {
+          enabled: true
+        }
+      }
+    })
+  })
+
+  afterAll(() => {
+    jest.resetAllMocks()
+  })
+
+  describe(`GET ${url} route`, () => {
+    test('returns 200', async () => {
+      const options = {
+        method: 'GET',
+        url,
+        auth
+      }
+
+      const res = await global.__SERVER__.inject(options)
+
+      expect(res.statusCode).toBe(200)
+      const $ = cheerio.load(res.payload)
+      expect($('h1 > label').text().trim()).toMatch('What is the vet\'s Royal College of Veterinary Surgeons (RCVS) number?')
+      expect($('title').text().trim()).toEqual('What is the vet\'s Royal College of Veterinary Surgeons (RCVS) number? - Annual health and welfare review of livestock')
+      expectPhaseBanner.ok($)
+    })
+
+    test('when not logged in redirects to defra id', async () => {
+      const options = {
+        method: 'GET',
+        url
+      }
+
+      const res = await global.__SERVER__.inject(options)
+
+      expect(res.statusCode).toBe(302)
+      expect(res.headers.location.toString()).toEqual(expect.stringContaining('https://tenant.b2clogin.com/tenant.onmicrosoft.com/oauth2/v2.0/authorize'))
+    })
+  })
+
+  describe(`POST ${url} route`, () => {
+    let crumb
+
+    beforeEach(async () => {
+      crumb = await getCrumbs(global.__SERVER__)
+    })
+
+    test('when not logged in redirects to defra id', async () => {
+      const options = {
+        method: 'POST',
+        url,
+        payload: { crumb, numberAnimalsTested: '123' },
```

Comment:
I know it doesn't effect the test but can you change numberAnimalsTested to vetRCVSNumber